.vitepress/sidebar.mts

```diff
@@ -9,20 +9,40 @@ export const sidebar: DefaultTheme.SidebarItem[] = [{
 			link: 'https://docs.meower.org/api/rest.html',
 		},
 		{
-			text: 'Cloudlink',
+			text: 'Cloudlink (websocket)',
```

Comment:
there's no reason to change this, everything still refers to it as cloudlink. we will eventually change the name when we start renaming all the services

Reply:
Ok, when I get to my computer I will change that.